go/service/chat_local.go

```diff
@@ -793,3 +933,42 @@ func (h *chatLocalHandler) assertLoggedIn(ctx context.Context) error {
 	}
 	return nil
 }
+
+// Sign implements github.com/keybase/go/chat/s3.Signer interface.
+func (h *chatLocalHandler) Sign(payload []byte) ([]byte, error) {
+	arg := chat1.S3SignArg{
+		Payload: payload,
+		Version: 1,
+	}
+	return h.remoteClient().S3Sign(context.Background(), arg)
+}
+
+func (h *chatLocalHandler) uploadAsset(ctx context.Context, sessionID int, params chat1.S3Params, local chat1.LocalSource, progress chat.ProgressReporter) (chat1.Asset, error) {
```

Comment:
I added a ticket for that.  CORE-3959.  The Multi api supports it, but I didn't write the code for resuming (yet).